packages/lab/src/__tests__/__e2e__/stepper-input/StepperInput.cy.tsx

```diff
@@ -214,6 +246,24 @@ describe("Stepper Input", () => {
     );
   });
 
+  it("does not decrement below the minimum value", () => {
+    cy.mount(<StepperInput defaultValue={-1} min={-1} />);
+
+    cy.findByRole("spinbutton").should("have.value", -1);
+
+    cy.findByLabelText("decrement value").realClick();
+    cy.findByRole("spinbutton").should("have.value", -1);
+  });
```

Comment:
This doesn't check whether `onChange` should be called.

Reply:
updated